cluster/common.sh

```diff
@@ -1100,6 +1104,12 @@ function generate-certs {
     mv "kubelet.pem" "pki/issued/kubelet.crt"
     rm -f "kubelet.csr"
 
+    # create the webhook client cert with the correct groups
```

Comment:
I would maybe try to make this clearer that this is the client cert which kube-apiserver will use to call out to any webhooks.